pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -59,6 +58,7 @@ def collect_args(basename):
     It writes the arguments to the build.log, and then delegates to the real
     native compiler or linker.
     """
+    TOOLSDIR = Path(common.get_make_flag("TOOLSDIR"))
```

Comment:
This loads `Makefile.env` which wouldn't be available if `pyodide-build` is installed from PyPi so we avoid to run that command at import time.